README.md

```diff
@@ -337,128 +375,136 @@ text_sensor: #Consider excluding these sensors from you Home Assistant database
+        data: [0x44, 0x6D] # "Power On Function" / "Play"
 
   - platform: template
     name: "Turn off Playback device 1"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 4
-        data: [0x36]
+        data: [0x36]       # "Standby"
 
   - platform: template
-    name: "Playback device 1 home button"
+    name: "Playback 1 Home"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 4
-        data: [0x44, 0x09]
+        data: [0x44, 0x09] # "Root Menu"
 
   - platform: template
-    name: "Playback device 1 select/ok"
+    name: "Playback 1 Select/OK"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 4
-        data: [0x44, 0x00]
+        data: [0x44, 0x00] # "Select"
 
   - platform: template
-    name: "Playback device 1 exit/back"
+    name: "Playback 1 Back/Exit"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 4
-        data: [0x44, 0x0D]
+        data: [0x44, 0x0D] # "Exit"
 
   - platform: template
-    name: "Playback device 1 play/pause"
+    name: "Playback 1 Play/Pause"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 4
-        data: [0x44, 0x44]
+        data: [0x44, 0x44] # "Play" / "Pause Toggle"
 
+  # --- Playback Device 2 (PlayStation 4 example) ---
   - platform: template
     name: "Turn on Playback device 2"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
-        destination: 8
-        data: [0x44, 0x6D]
+        destination: 8     # Typical address for second playback device (PS4)
+        data: [0x44, 0x6D] # May vary depending on firmware
 
   - platform: template
     name: "Turn off Playback device 2"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 8
         data: [0x36]
 
   - platform: template
-    name: "Playback device 2 play/pause"
+    name: "Playback 2 Play/Pause"
     on_press:
       hdmi_cec.send:
-        # "source" can optionally be set, like if you want to spoof another device's address
         destination: 8
         data: [0x44, 0x46]
+
 ```
 
+---
+## 3D printed case (ESP32-C3 SuperMini)
+
+If you’re using an ESP32-C3 SuperMini, you can 3D-print a dedicated case designed by [DIYtechie on MakerWorld](https://makerworld.com/en/models/1488957-esp32-c3-hdmi-case#profileId-1761813).
+
+![ESP32-C3 HDMI case](https://github.com/DIYtechie/resources/blob/master/images/ESP32%20C3%20HDMI%20CASE.jpg?raw=true)
```

Comment:
This image takes up a lot of space, but we can leave it there. If others want to contribute their own cases to the README, i'll have to remove it or you can propose a smaller design.